Loop/Managers/LoopAppManager.swift

```diff
@@ -259,20 +262,20 @@ class LoopAppManager: NSObject {
     }
 
     private var rootViewController: UIViewController? {
-        get { window?.rootViewController }
-        set { window?.rootViewController = newValue }
+        get { windowProvider?.window?.rootViewController }
+        set { windowProvider?.window?.rootViewController = newValue }
     }
 }
 
 // MARK: - AlertPresenter
 
 extension LoopAppManager: AlertPresenter {
     func present(_ viewControllerToPresent: UIViewController, animated: Bool, completion: (() -> Void)?) {
-        rootViewController?.present(viewControllerToPresent, animated: animated, completion: completion)
+        rootViewController?.topmostViewController.present(viewControllerToPresent, animated: animated, completion: completion)
```

Comment:
yeah my first instinct was that this was the only thing missing, but alas after I tried it it didn't fix the problem.  Thanks for sussing it out!

Reply:
True, will update, not that it really matters since both will be around forever. :)